server/src/http_server/pages/blog/md/html.rs

```diff
@@ -287,13 +291,25 @@ impl IntoHtml for Emphasis {
 }
 
 impl IntoHtml for Image {
-    fn into_html(
-        self,
-        _config: &AppConfig,
-        _context: &SyntaxHighlightingContext,
-    ) -> Result<Markup> {
+    fn into_html(self, config: &AppConfig, context: &SyntaxHighlightingContext) -> Result<Markup> {
+        let article_path = context
+            .current_article_path
+            .as_deref()
+            .unwrap_or("unknown")
+            .split('/')
+            .next()
```

Comment:
Would be nice to not do any path parsing 'manually' 🤔 

This is grabbing the stuff before the first `/` right? Whats after the slash lol?

Reply:
after the slash there's a article name that's not really needed for the `image` path

in my local directory `http://localhost:3000/posts/BattlesnakeMinimax/Minimax%20in%20Battlesnake/`

In this I grab everything before `Minimax%20in%20Battlesnake` that's why I split it